pinot-controller/src/main/java/com/linkedin/pinot/controller/helix/core/PinotTableIdealStateBuilder.java

```diff
@@ -327,7 +329,7 @@ public Boolean call() throws Exception {
         _exception = e;
         throw e;
       } finally {
-        IOUtils.closeQuietly(consumerWrapper);
+        consumerWrapper.close();
```

Comment:
better to force the wrapper to implement closeable and use IOUtils to close it. If not, we will have to handle the exceptions from close method